SoftLayer/CLI/virt/os_available.py

```diff
@@ -0,0 +1,28 @@
+"""Get the all Os available."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@environment.pass_env
+def cli(env):
+    """Get the all Os available"""
+
+    table = formatting.KeyValueTable(['id', 'keyName', 'Description', 'price', 'setup fee'])
```

Comment:
The key values are better in UpperLower case letters
```suggestion
    table = formatting.KeyValueTable(['Id', 'KeyName', 'Description', 'Price', 'Setup Fee'])
```